plugins/logs/commands

```diff
@@ -11,34 +12,60 @@ case "$1" in
     APP_ROOT="$DOKKU_ROOT/$APP"
     COLORS=(36 33 32 35 31)
 
-    if (is_deployed "$APP"); then
+    if ! (is_deployed "$APP"); then
+      echo "Application's container not found"
+      exit 1
+    fi
+
+    shift 2;
+    TEMP=$(getopt -o htqn:p: --long help,tail,quiet,num:,ps: -n 'dokku logs' -- "$@")
+    if [[ $? != 0 ]]; then usage >&2 ; exit 1 ; fi
+    eval set -- "$TEMP"
+
+    LINE_NUMBERS="100"
+    while true; do
+        case "$1" in
+            -h|--help) usage; exit 0 ;;
+            -t|--tail) DOKKU_LOGS_ARGS+="--follow "; shift ;;
+            -n|--num) LINE_NUMBERS="$2"; shift 2 ;;
+            -p|--ps) ONLY_PROCESS="$2"; shift 2 ;;
+            -q|--quiet) NO_PRETTY_PRINT="true"; shift ;;
```

Comment:
I'm concerned with polluting the global variable space here. We (and I mean that to include me) have not been very good at this and have found ourselves causing unintended bugs in the past because of it.

Perhaps we can prepend these variable with `DOKKU_LOGS_*` or, per the [style guide](https://github.com/progrium/bashstyle), turn this into a function and use `local` variable declarations?